### DISCUSSION TITLE
Compile error on PlatformIO with Arduino Core 2.0.1

### DISCUSSION THREAD

**Discussion by Speedy2k** [Makuna/NeoPixelBus#531] [Q&A]
I am trying to compile my program with Arduino Core 2.0.1 and i get this error message even with your latest release!

`In file included from C:/Users/Martin/.platformio/packages/framework-arduinoespressif32/tools/sdk/esp32/include/driver/include/driver/i2s.h:16:0,
                 from .pio/libdeps/esp32_dev/NeoPixelBus/src/internal/Esp32_i2s.c:52:
C:/Users/Martin/.platformio/packages/framework-arduinoespressif32/tools/sdk/esp32/include/hal/include/hal/i2s_types.h:95:31: error: expected ',' or '}' before '__attribute__'
     I2S_COMM_FORMAT_I2S       __attribute__((deprecated)) = 0x01, /*!< I2S communication format I2S, correspond to `I2S_COMM_FORMAT_STAND_I2S`*/
                               ^
cc1.exe: warning: unrecognized command line option '-Wno-frame-address'
*** [.pio\build\esp32_dev\lib71d\NeoPixelBus\internal\Esp32_i2s.c.o] Error 1`

When i compile for 1.0.6 it is OK!

**Comment by Makuna** [comment]
@Speedy2k NOT AN ISSUE means a discussion, of which there is a completely different area for.  I moved it over for you.

I just tried ESP32 Board support v2.0 (2.0.1 is not released) and it builds fine within Arduino IDE.
Further, the error is not in NeoPixelBus, its in i2s_types.h.  Its almost like what ever compiler you are using doesn't support __attribute__.
